src/algokit/cli/tasks/mint.py

```diff
@@ -103,26 +194,28 @@ def _validate_asset_name(context: click.Context, param: click.Parameter, value:
     prompt="Provide the address or alias of the asset creator",
     help="Address or alias of the asset creator.",
     type=click.STRING,
+    callback=run_callback_once(callback=_get_creator_account),
```

Comment:
```suggestion
    callback=run_callback_once(_get_creator_account),
```
Reads nicer and is more consistent.